docs/11-persistence.md

```diff
@@ -19,35 +19,35 @@ The structure of the data stored on the device is:
 ```
 
 The `cities-meta.json` contains information about the last content update:
-```
+```json
 {  
-   "regensburg":{  
-      "languages":{  
-         "de":{  
-            "lastUpdate":"2019-08-12T00:03:19.457Z"
+   "regensburg": {  
+      "languages": {  
+         "de": {  
+            "last_update": "2019-08-12T00:03:19.457Z"
```

Comment:
Why did you choose to use underscore here? In the cities.json, we use camelCase it seems :thinking:

Reply:
We use underscores everywhere here except in cities.json. I don't care if we use underscore or camelCase, but since the cities.json was the only with camelCase I chose underscores. Adjusted cities.json though.